editor/example/doc-examples/area.ts

```diff
@@ -0,0 +1,44 @@
+export const AREA = `{
```

Comment:
> yes, I see the drawbacks of missing type-checking. The main reason I am using the string specs is a) to keep the comments; b) these examples are previously stored in gist and more like strings. Maybe I can run an additional spec validation function for these string specs to address your concerns?

One thing I really like in my IDE is that renaming properties in the schema is very simple, i.e., when I rename something, all properties used in other files are updated at once. But, having string-format examples will make it difficult to find corresponding changes in the examples and to update them.

If you think comments are something that is really important for describing examples, I think we can just keep them as-is (i.e., string) and revisit this in the future. But, if not, let's change these examples to JSON objects so that we can more efficiently maintain examples in this repo. It's your call!

Reply:
Well, in this case, I think an object is better than a string for code maintenance.  No problem, I will change them to JSON objects